pkg/commands/resolve.go

```diff
@@ -66,6 +66,9 @@ func addResolve(topLevel *cobra.Command) {
 			if err != nil {
 				return fmt.Errorf("error creating builder: %w", err)
 			}
+			if bo.DockerRepo != "" {
+				po.DockerRepo = bo.DockerRepo
+			}
```

Comment:
Bleh, this also doesn't feel right. Sorry to keep moving things around on you, I'm struggling to find a good way to express this without having multiple copies of this floating around.

Reply:
What about creating an overrider function to set and pass some variables from _builder_ to _publisher_ struct?

```golang
makeOverrider(bo, po)
```

I'm not sure if `overrider` is the right word but used it as a placeholder.